setup.py

```diff
@@ -28,13 +28,13 @@ def run_tests(self):
       version='0.1',
       description='',
       url='http://github.com/ctk3b/misibi',
-      author='Christoph Klein, Timothy C. Moore',
-      author_email='christoph.klein@vanderbilt.edu, timothy.c.moore@vanderbilt.edu',
+      author='Christoph Klein, Timothy C. Moore', 'Davy Yue'
```

Comment:
This field is typically used for the official package maintainers (i.e. people who approve PRs and distribute the package via PyPI/conda). We're more than happy to add you once you get to that point but for now you fall into the "contributor" category :)

One thing you could add is module doc strings to the entire project [like this](https://github.com/mdtraj/mdtraj/blob/master/mdtraj/geometry/order.py#L1-L21). That would provide an intermediate level of authorship acknowledgement between maintainers and the github contribution list